Yeah, hardwiring these versions is ugly. Let me know if you know a
better way.

--- soprano-2.3.70/CMakeLists.txt.orig	2009-11-29 10:04:17.000000000 +0100
+++ soprano-2.3.70/CMakeLists.txt	2010-02-02 00:40:19.000000000 +0100
@@ -64,6 +64,8 @@
 endif(NOT SOPRANO_DISABLE_CLUCENE_INDEX)

 if(NOT SOPRANO_DISABLE_SESAME2_BACKEND)
+  set(JAVA_INCLUDE_PATH /usr/lib/gcc/i686-frugalware-linux/4.5.2/include/)
+  set(JAVA_JVM_LIBRARY /usr/lib/libjvm.so)
   find_package(JNI)
   if(JAVA_INCLUDE_PATH AND JAVA_JVM_LIBRARY)
     file(READ ${JAVA_INCLUDE_PATH}/jni.h jni_header_data)

